compose-files/extra-services/bitcoin.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
```

Comment:
why version 3.8? is there something present in 3.8 vs 3?

Reply:
I'm not looking for the oldest version that works, but a current stable version that works and will be compatible with future versions.
3.8 seems like a good balance for me. But if you prefer 3, I'm sure it will also work fine also.

Reply:
I have no problem with using 3.8 per se, but if it's used for this compose file it should also be used everywhere. typically you only use compose point releases is if there is a specific syntax/feature you're trying to use. else, it's more common to use `2` or `3`. in this repo, it was setup for version 3, since the point release didn't have a compelling use (at the same time, `3` is compatible with older version of docker). 
https://docs.docker.com/compose/compose-file/compose-versioning/#version-38

that said, if you find some interesting features available that outweigh the benefit of working with older docker versions, `3.8` is perfectly fine - just be sure to update all of the compose files.

Reply:
additionally, version 2 of compose still *works* fine - you're just not going to have access to the features in version 3 of the format.

Reply:
I completely agree that we must be coherent and all files should use the same version number. I was going to switch it to version 3, but reviewing the features per version, I see that [version 3.5](https://docs.docker.com/compose/compose-file/compose-versioning/#version-35) introduced the `name for networks, secrets and configs`, which funny enough we are using both in my new files and in your old file that use version 3.
Version 3.5 requires Docker Engine version 17.09
So, I'm changing all files to version 3.5 and adding the Docker Engine minimum version to the requirements docs.
If for whatever reason you still prefer v.3 we can always switch them all again, but at least they will always all have the same version.